File: src/main/resources/static/js/service/regionDeviceTypeService.js
app.service('regionDeviceTypeService',function ($http) {
    this.findAll = function () {
        return $http.get('../regionDeviceType/findAll');
    };

    this.findByUserId = function (uid) {
        return $http.get("../regionDeviceType/findByUserId?uid="+uid);
    };

    this.findByCurrentUser = function () {
        return $http.get("../regionDeviceType/findByCurrentUser");
    };
});

File: src/main/java/com/shengxuan/speed/service/RegionDeviceTypeService.java
package com.shengxuan.speed.service;

import com.shengxuan.speed.entity.RegionDeviceType;

import java.util.List;

public interface RegionDeviceTypeService {
    List<RegionDeviceType> findByUserId(Integer uid);

    List<RegionDeviceType> findByCurrentUser();
}
